COLLABORATOR_GUIDE.md

```diff
@@ -282,7 +282,6 @@ providing a Public API in such cases.
   * Resolving critical security issues.
   * Fixing a critical bug (e.g. fixing a memory leak) requires a breaking
     change.
-  * There is TSC consensus that the change is required.
```

Comment:
If the authority was delegated to the Release WG, then shouldn't this be `s/TSC/Release WG/`?

We have policies so that there is predictability and  transparency to the decisions made by the responsible people, but those people ultimately have the right to break those policies, if as a group they decide its the right thing to do. The planned openssl 1.1.1 update in the 10.x release line for example best fits the bullet being removed here.

Reply:
@sam-github Yes and no. I'd say it's a bigger change than that. All these bullet points should probably be replaced with a statement that what ends up in LTS/Current is up to Release WG, with a link to where Release WG documents their requirements. I'm doing things iteratively, though, because small and focused changes to this document seem to be things we can talk about and land, but big changes run a bigger risk of being stalled. (Not sure if that was my experience with this doc or elsewhere, but it's definitely A Thing.)

So my plan is: First, remove this bullet point because it's wrong. *Then* replace the remaining stuff with a link to Release WG material--probably https://github.com/nodejs/Release#release-plan but maybe there's something even better somewhere?  And there may be other steps too. But those are the next two. :-D